packages/babel-parser/test/fixtures/experimental/explicit-resource-management/invalid-using-binding-pattern-declaration/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "throws": "Unexpected token (2:16)"
```

Comment:
Could we continue parsing `using [no LineTerminator here] {` with the old logic, to provide the good error message, and only use the new logic for `using [`?

Reply:
`using { x } = 1` would be a valid lhs assignment under [the extractors proposal](https://github.com/tc39/proposal-extractors). In the future if we implement that proposal, it will throw the missing plugin error. So I removed the recoverable error here.